src/utils/file-upload-utils.js

```diff
@@ -14,8 +14,10 @@ const ALLOWED_FILE_EXTENSIONS = [
   "pdf",
   "png",
   "jpg",
+  "jpeg",
```

Comment:
Hey just a heads up, we maintain this list over in the FE, may I trouble you to add in a corresponding PR in the FE for this?

Reply:
@kishore03109 added here: https://github.com/isomerpages/isomercms-frontend/pull/1413